src/hooks/bet/usePotentialPayout.ts

```diff
@@ -34,15 +34,18 @@ const usePotentialPayoutFromContract = ({
     [odds]
   );
   const { data: bnPotentialPayout } = useContractRead({
-    addressOrName: marketAddress,
-    contractInterface: marketContractJson.abi,
+    address: marketAddress,
+    abi: marketContractJson.abi,
     functionName: "getPotentialPayout",
     args: [b32PropositionId, bnWager, bnOdds]
   });
 
   const potentialPayout = useMemo(() => {
     if (!bnPotentialPayout) return "0";
-    return ethers.utils.formatUnits(bnPotentialPayout, tokenDecimal);
+    return ethers.utils.formatUnits(
+      bnPotentialPayout as BigNumber,
```

Comment:
would `bnPotentialPayout` not already by a BigNumber?

Reply:
unfortunately not :(
<img width="506" alt="Screen Shot 2022-11-10 at 2 41 49 pm" src="https://user-images.githubusercontent.com/10498040/201002376-8c450f38-8f8e-4599-8040-b1dbe38adfc7.png">

Reply:
does `useContractRead` accept a generic?